### ISSUE TITLE
Python docs build fails and hangs in CI

### ISSUE THREAD

**Issue by ptrendx** [apache/incubator-mxnet#16624]
## Description
Python docs step fails with error

```
Sphinx parallel build error:
nbsphinx.NotebookError: TemplateNotFound in tutorials/deploy/export/onnx.ipynb:
rst.tpl
```

and then hangs (job is killed after 3 hours time out). The commit to master branch that started failing did not touch documentation at all.

@aaronmarkham 

## Occurrences
From current master: http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Fwebsite/detail/v1.6.x/3/pipeline/85
From several PRs:
http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Fwebsite/detail/PR-16615/7/pipeline
http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Fwebsite/detail/PR-16619/3/pipeline

**Comment by ChaiBapchya** [comment]
@aaronmarkham 
Another failure

http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Fwebsite/detail/PR-16570/8/pipeline

**Comment by aaronmarkham** [comment]
Most recent change to onnx.md:
https://github.com/apache/incubator-mxnet/commit/06ce371b54a51c58e2cf1bfb93f1f9b2c568c364#diff-e8d28d7ac094c6a300e2f0ec574ea2c5
It's a broken link fix and I don't see how this would be the issue.

However, I did see the parallel build issue come up when I was building locally. I added a patch to detect the number of processors.
https://github.com/apache/incubator-mxnet/pull/16512

@ThomasDelteil suggested that there was probably some other issue (maybe related to the tutorial conversion from md to ipynb) and that all I did was kick the can. Yes, I found and fixed an issue where the build had a fixed number of procs, but the Sphinx errors were probably something else.

But, I don't know what it is. Pinging Thomas to see if he has ideas.
